src/app/components/forms/community/details/details.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from 'vue-property-decorator';
+import { Community } from '../../../../../_common/community/community.model';
+import { BaseForm } from '../../../../../_common/form-vue/form.service';
+
+@Component({})
+export default class FormCommunityDetails extends BaseForm<Community> {
+	modelClass = Community;
+	saveMethod = '$saveDetails' as '$saveDetails';
+
+	get communityUrl() {
+		return (
+			'gamejolt.com/c' +
+			'/<b>' +
```

Comment:
What's with this weird new line? tslint?

Reply:
yes. the thing we use for spell checking.